tests/apollo/test_skvbc_block_accumulation.py

```diff
@@ -51,9 +48,6 @@ def start_replica_cmd(builddir, replica_id, view_change_timeout_milli="10000"):
             "-i", str(replica_id),
             "-s", status_timer_milli,
             "-v", view_change_timeout_milli,
-            "-b", BATCH_BY_REQ_NUM,
-            "-q", str(MAX_REQ_NUM_IN_BATCH),
-            "-z", BATCH_FLUSH_PERIOD,
```

Comment:
Why remove this ability?

Reply:
This was added initially assuming count could be precisely calculated. Since that is not the case removed them as existing batch test suite configuration. Also it is observed to execute faster.